### PR TITLE
Ngrsge advanced

### PR DESCRIPTION
**Pull request by NGRhodes**
Some old examples that have been requested to be retained.

### CODE REVIEW

**Review by jhodrien — COMMENTED** [review]
Can we have consistency in the way we represent bits that need filling in?  You've used $something and <something> in different places.   Other pages I think have gone with <something>.

Can we also have the bash blocks as bash blocks, and not inline preformatted, and stick a $ before the command to show it's a shell prompt?

**Review by jhodrien — COMMENTED** [review]
Sorry, missed on first review.  Need to work out what you're doing with all the numbers, as this is written for ARC2, with 16 cores per node.  Either needs to be tweaked to related to ARC3/4, or make clear that it's talking about a hypothetical cluster with 16 cores per node.

**Comment by NGRhodes** [comment]
@jhodrien How does it look now ?

**Comment by jhodrien** [comment]
Final pedantry now, and I'm utterly failing at just adding this as a suggested commit:

"There are 24 cores per node on ARC3 , you"
"There are 40 cores per node on ARC4 , you"

Should that be:

"There are 24 cores per node on ARC3, so you"
...

jh


--
John Hodrien
Research Software Engineer, Research Computing
10.13 IT Services Building, University of Leeds, 0113 34 35471

________________________________

**Comment by NGRhodes** [comment]
There you go @jhodrien :)

**Review by jhodrien — APPROVED** [review]

File: book/usage/advanced.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```bash
+#$ -V 
+#$ -cwd
+#$ -l hr_t=01:00:00 
+#$ -l nodes=8, ppn=10, tpp=4 
+mpirun ./mixed.exe
+```
+Note that the `OMP_NUM_THREADS` environment variable is automatically set by the batch system and so you do not need to set this in your environment.
+
+## Job Dependencies
```
Comment (Sparrow0hawk) [inline book/usage/advanced.md:57]: This section could link to the `-hold_jid` section in this [table](https://arcdocs.leeds.ac.uk/usage/batchjob.html?highlight=hold_jid#list-of-sge-options)

**Review by Sparrow0hawk — APPROVED** [review]
Minor suggestion but otherwise approve

File: book/usage/advanced.md (hunk 1 above)
Comment (NGRhodes) [reply book/usage/advanced.md:57]: Done

**Pull request merged**